---
title: Research in the Depledge Lab
layout: default
group: research
---

<div class="row">

<h2><code style="color : orangered"><b>This page is under construction</b></code></h2>


<br />
<br />

### Research in the Depledge lab focuses on several broad themes:

<br />

#### (1) Regulation of the varicella zoster virus (VZV) latent-lytic switch in sensory neurons

We have previously shown that VZV transcription in latently infected human sensory ganglia is generally restricted to the [VZV latency transcript (VLT)](https://www.nature.com/articles/s41467-018-03569-2). The latent isoform of VLT comprises five exons and putatively encodes a protein (pVLT) although it is now known whether this is translated in latent infections. In iPSC-derived sensory neuronal models of VZV latency, we have since demonstrated that reactivation requires an alteration to 3'end processing in the VLT locus that results in alternative splicing and the addition of a sixth exon that encodes the VZV immediate-early transactivator ORF63. This novel transcript, termed [VLT63](https://www.nature.com/articles/s41467-020-20031-4), is required for transactivation of the viral genome during reactivation from latency. Our efforts are now focused on identifying and understanding the molecular mechanisms that regulate 3' end processing in the VLT locus.

<img class="img-fluid" src="/static/img/latreacmodel.jpg" alt="overview of nanopore direct RNA sequencing">

<br />


#### (2) Epitranscriptomic regulation of herpesvirus infections 



#### (3) A systems approach to interrogating herpesvirus biology, one RNA at a time

The biology of herpesviruses is informed by their capacity to encode multitudes of diverse proteins and regulatory RNAs within highly compact double stranded DNA genomes, most of which are refractory to detailed analyses using traditional short read sequencing approaches. The recent emergence of nanopore direct RNA sequencing ( methodologies has enabled, for the first time, the ability to directly sequence full length poly(A) RNAs synthesized within the cellular environment, preserving all original features such as chemically modified nucleotides and poly(A) tails. Initial applications of the DRS methodology enabled [our lab](https://www.nature.com/articles/s41467-019-08734-9) and others to generate high resolution herpesviral transcriptome annotations have significantly expanded on the numbers of coding and non coding RNAs produced during infection of diverse cell types. We have since furthered these applications to include profiling of N6-methyladenosine (m6A) installation on [HSV](https://www.pnas.org/doi/10.1073/pnas.2104805118) and VZV transcriptomes during productive infections This was made possible through our development of complementary computational frameworks [DRUMMER](https://github.com/DepledgeLab/DRUMMER) and [NAGATA](https://github.com/DepledgeLab/NAGATA) that leverage genome level DRS alignments to produce high resolution viral transcriptome annotations that can subsequently be used as a basis for the study of [RNA modifications](https://academic.oup.com/bioinformatics/article/38/11/3113/6569078) and/or analyses of RNA expression changes and poly(A) tail lengths.

<img class="img-fluid" src="/static/img/drs.jpg" alt="schematic overviews of DRUMMER and NAGATA">

<br />




<!-- 

##### (1) Characterizing the roles of diverse base modifications (including m6A) on host and viral RNAs during productive infections

##### (2) Investigating the functions of herpesviral latency-associated transcripts and other non-coding RNAs (including miRNAs) during both lytic and latent infections

##### (3) Characterizing the transcriptional complexity of diverse viruses (herpesviruses, adenoviruses, coronaviruses) in distinct cell types, such as epithelial cells and neurons.

**This is achieved through the integration of computational biology with a variety of high-throughput molecular techniques to study multiple aspects of virus-host interactions on a genome-wide scale. Specific targets of interest are subsequently followed up using classical molecular biology approaches.**

-->

</div>
